server/dba/seeders/prod/01-christmasTreesForests.js

```diff
@@ -10,8 +10,8 @@ module.exports = {
         tree_height: 0,
         stump_height: 6,
         stump_diameter: 6,
-        start_date: '2017-11-01T12:00:00Z',
-        end_date: '2018-01-06T12:00:00Z',
+        start_date: '2018-11-01T12:00:00Z',
```

Comment:
does this mean that the db needs to be blown up, or will it override?

Reply:
@lauraGgit currently, christmas trees related lookup tables (forests, species, locations) data gets reset on every build (server/Procfile).